muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json

```diff
@@ -0,0 +1,1905 @@
+      "statusChangeError": "Message status change failed. Please try again.",
+      "updateError": "$t(common:notifications.updateError)",
+      "updateError_message": "Message editing failed. Please try again.",
+      "updateError_announcement": "Announcement editing failed. Please try again.",
+      "updateError_contactLog": "Contact log editing failed. Please try again.",
+      "updateError_label": "Updating the label failed. Please try again.",
+      "updateError_signature": "Updating the signature failed. Please try again.",
+      "updateError_discussionArea": "Updating the discussion area failed. Please try again.",
+      "updateSuccess": "$t(common:notifications.updateSuccess)",
+      "updateSuccess_message": "Updating the message failed. Please try again.",
+      "updateSuccess_announcements": "Upading the announcement failed. Please try again.",
+      "updateSuccess_contactLog": "Updating the contact event failed. Please try again.",
+      "workspacesRequiredError": "Message has no target courses"
+    },
+    "validation": {
+      "beginDate": "Announcement begin date is mandatory",
+      "caption": "$t(common:validation.caption)",
+      "caption_message": "Message caption is mandatory.",
+      "caption_announcements": "Announcement title is mandatory",
+      "content": "$t(common:validation.content)",
+      "content_message": "Message content is mandatory",
+      "content_announcements": "Announcement is mandatory",
+      "endDate": "Announcement end date is mandatory",
+      "name": "$t(common:validation.name)",
+      "name_area": "Area name is mandatory",
+      "name_labels": "Label name is mandatory",
+      "recipients": "Message recipient is mandatory",
+      "targetGroup": "Announcement target group is mandatory"
+    },
+    "wcag": {
+      "content": "Message content",
+      "content_truncated": "Truncated message content",
+      "currentPage": "Current page",
+      "date": "Message date",
+      "goToPage": "Go to page",
+      "label": "Message label",
+      "next10": "Move 10 pages forwards",
+      "OPUserAvatar": "Discussion opening poster avatar",
+      "pager": "Discussion paging functionalities",
+      "prev10": "Move 10 pages backwards",
+      "recipients": "Message recipients",
+      "replyCount": "Message reply count",
+      "setSelected": "Set message selected",
+      "send": "Send message",
+      "sender": "Message sender",
+      "selectArea": "Selection of the discussion area",
+      "unread": "Unread message"
+    }
+  },
+  "tasks": {
+    "actions": {
+      "approve": "Mark approved",
+      "cancel": "Cancel approval request",
+      "done": "Mark done",
+      "incomplete": "Mark incomplete",
+      "send": "Send for approval"
+    },
+    "content": {
+      "empty": "$t(common:content.empty)",
+      "empty_tasks": "No tasks",
+      "empty_student": "There are no tasks given for you and you have created any. You can create new tasks in the studies view.",
```

Comment:
"There are no tasks given for you and you haven't created any. You can create new tasks in the studies view."